lib/registries/addon/overview/template.hbs

```diff
@@ -55,6 +58,25 @@
             </gutters.left-gutter>
 
             <gutters.body local-class="Body">
+                {{#if this.showMobileView}}
+                    <div local-class="SocialDropdown">
+                        <div data-gutter-closed="{{this.socialDropdownClosed}}" local-class="Right Gutter">
```

Comment:
We should leave the gutter styles in the gutters component. Seems like this should use `<ResponsiveDropdown>`?